vulkan/source/gfx/vulkan/package.d

```diff
@@ -147,6 +147,10 @@ VulkanInstance createVulkanInstance(VulkanCreateInfo createInfo=VulkanCreateInfo
     import std.range : chain;
     import std.string : toStringz;
 
+    // Surface instance extensions for the configured surface integration are mandatory so that
+    // the application may create surfaces
+    createInfo.mandatoryExtensions ~= surfaceInstanceExtensions;
```

Comment:
When configured with glfw, `surfaceInstanceExtensions` cannot be evaluated at compile-time. Also, surface extensions are mandatory for applications to function

Reply:
You're right, it is better at runtime anyway.
That way we can determine whether Wayland is available on the system